### ISSUE TITLE
(Mining)Sites need to be its own entity

### ISSUE THREAD

**Issue by davidmihalyi** [NRGI/rp-org-frontend#5]
These two big entity types need to be added, but probably for next iteration.

**Comment by byndcivilization** [comment]
Why are these independent entities. They seem like basically the same thing (a tract of land under development) that are separated by commodity type. Can we model them independently with a field that maps either commodity or extraction type?

**Comment by davidmihalyi** [comment]
would rather keep them seperate
1. oil field is a key entity (or mine deposit for mining but deposit level data is very uncommon)  
2. mine site is a  key entity (or oil platform but again uncommon) 

both  are often equivalent to a project

**Comment by davidmihalyi** [comment]
ACtually let me rephrase:
I don`t mind them been modeled equivalent, as long as you can clearly label it.

**Comment by davidmihalyi** [comment]
so if you allow user to filter field level data, that is fine. (and not show site level data)

**Comment by byndcivilization** [comment]
Can we list the metadata we want associted with each type here? We can see if there's some overlap

**Comment by davidmihalyi** [comment]
the metadata is the same. 
Name
Address
Country (oil fields may be cross border)
Lat +Long 
We also want to map them with source to proj/concess/contracts 
Would be also great to be able to  say that some of our production or reserve data is collected at field or site level (rather than just saying it relates to project A).

**Comment by davidmihalyi** [comment]
See glossary on context: 
http://alpha.resourceprojects.org/glossary.html

Oil or gas field: An area of land or seabed with a hydrocarbons reservoir or multiple geologically related reservoirs beneath, especially in amounts that justify commercial exploitation. When oil or gas fields are discovered, they are often given a name, sometimes following the name on the well with which the discovery was made. Oil and gas projects may aim to recover oil and/or gas from one or multiple connected fields.

Mine site: Places or areas from which commercial minerals are or were removed by excavation. A project would typically include one or multiple closely linked mine sites.

Again, Chris, i`m fine as modelling as one, if we have an additional variable to ensure we know what is a field and what is a site to filter in between.

**Comment by byndcivilization** [comment]
dont we want to store these as polygons rather than or in addition to lat long?

**Comment by davidmihalyi** [comment]
We have not seen datasets yet where these are polygons, all we have is lat and lon

concession is the one that is often polygon shaped in datasets.

**Comment by byndcivilization** [comment]
Okay we have a model implemented:

```
{
    site_name: String,
    field: Boolean,
    site_type: [fact],
    site_aliases: [{
        type: ObjectId,
        ref: 'Alias'}],
    site_established_source: source,
    site_address: [fact],
    site_country: [fact],
    site_coordinates: [fact],
    description: htmlSettings
}
```

I'm using a "field" bolean to flag the difference of records since theyre basically the same data structure.

**Comment by mattfullerton** [comment]
So with this implemented I will create sites/fields also at the import stage. I assume we should now stop storing multiple location facts for projects and instead store sites, yes?

**Comment by davidmihalyi** [comment]
one site/field - one location is more appropriate way to do it.
a project then can have multiple sites/fields...

need to add the site/field boolean to google template, coz it is not there right now.
